templates/pioneer-core/demographic-survey/demographic-survey.1.schema.json

```diff
@@ -49,7 +49,7 @@
     "origin": {
       "type": "object",
       "properties": {
-        "hispanicLatinoSpanish": {
+        "hispanicLatinxSpanish": {
```

Comment:
The old field needs to be retained, with a comment that marks it as deprecated. This does not follow schema evolution rules. You probably also want to update the validation document for coverage.

Reply:
Thanks for the feedback!